src/main/java/frc/robot/commands/GoHomeSequence.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.ConditionalCommand;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.subsystems.conveyer.Conveyer;
+import frc.robot.subsystems.elevator.Elevator;
+import frc.robot.subsystems.elevator.Elevator.ElevatorPosition;
+import frc.robot.subsystems.intake.Intake;
+import frc.robot.subsystems.intake.Intake.IntakePosition;
+
+public class GoHomeSequence extends SequentialCommandGroup {
+  public GoHomeSequence(Intake intake, Elevator elevator, Conveyer conveyor) {
+    addRequirements(elevator, conveyor, conveyor);
+    addCommands(
+        new ConditionalCommand(
+            new SequentialCommandGroup(
+                new InstantCommand(() -> intake.moveToPos(IntakePosition.STOWED)),
```

Comment:
Make `moveToPos` private, add `setDesiredPosition` function, and call that here instead